src/index.js

```diff
@@ -71,12 +67,12 @@ if (program.sam) {
 }
 AWS.config.credentials = getAWSCredentials();
 AWS.config.region = program.region;
-const cloudformation = new AWS.CloudFormation();
-const cloudwatch = new AWS.CloudWatch();
-const cloudwatchLogs = new AWS.CloudWatchLogs();
-const eventBridge = new AWS.EventBridge();
-const schemas = new AWS.Schemas();
-const lambda = new AWS.Lambda();
+let cloudformation = new AWS.CloudFormation();
```

Comment:
this is showing this while file should be a class with attributes for these sort of changes